test/helpers/uid-generator.js

```diff
@@ -0,0 +1,18 @@
+var lastId = 0;
+
+function nextUid() {
+  lastId++;
+  return lastUid();
+}
+
+function lastUid() {
+  return '' + lastId;
+}
+
+function resetUidCounter() {
+  lastId = 0;
+}
+
+module.exports = nextUid;
+module.exports.last = lastUid;
+module.exports.reset = resetUidCounter;
```

Comment:
I also think something is weird about this file. The way it's coded is when you require it, nextUid is called seems like magic to me. This file should be simplified to:

```
var lastId = 0;

function nextUid() {
  lastId++;
  return lastUid();
}

function lastUid() {
  return '' + lastId;
}

function resetUidCounter() {
  lastId = 0;
}

exports.next = nextUid;
exports.last = lastUid;
exports.reset = resetUidCounter;
```